edx_when/models.py

```diff
@@ -93,13 +93,18 @@ class ContentDate(models.Model):
     field = models.CharField(max_length=255, default='')
     active = models.BooleanField(default=True)
     block_type = models.CharField(max_length=255, null=True)
+    assignment_title = models.CharField(max_length=255, blank=True, default='', db_index=True)
+    course_name = models.CharField(max_length=255, blank=True, default='')
+    subsection_name = models.CharField(max_length=255, blank=True, default='', db_index=True)
```

Comment:
As far as I know, OeX tries to avoid using names such as section, subsection, etc. I suggest using component or block. What do you think?